.gitlab-ci/docker-ros.yml

```diff
@@ -219,19 +227,19 @@ Test dev-arm64:
       when: never
   script:
     - |-
-      if [[ ${PLATFORM} =~ amd64 && ${PLATFORM} =~ arm64 ]]; then
-        [[ ${TARGET} =~ dev ]] && docker manifest create ${IMG_DEV} --amend ${_IMAGE_DEV_CI_AMD64} --amend ${_IMAGE_DEV_CI_ARM64}
-        [[ ${TARGET} =~ run ]] && docker manifest create ${IMG_RUN} --amend ${_IMAGE_RUN_CI_AMD64} --amend ${_IMAGE_RUN_CI_ARM64}
-      elif [[ ${PLATFORM} == amd64 ]]; then
-        [[ ${TARGET} =~ dev ]] && docker manifest create ${IMG_DEV} --amend ${_IMAGE_DEV_CI_AMD64}
-        [[ ${TARGET} =~ run ]] && docker manifest create ${IMG_RUN} --amend ${_IMAGE_RUN_CI_AMD64}
-      elif [[ ${PLATFORM} == arm64 ]]; then
-        [[ ${TARGET} =~ dev ]] && docker manifest create ${IMG_DEV} --amend ${_IMAGE_DEV_CI_ARM64}
-        [[ ${TARGET} =~ run ]] && docker manifest create ${IMG_RUN} --amend ${_IMAGE_RUN_CI_ARM64}
+      if [[ "${PLATFORM}" =~ amd64 && "${PLATFORM}" =~ arm64 ]]; then
```

Comment:
Why is the quoting necessary? Do we need to quote amd64/arm64 as well?

Reply:
Its just more robust and can handle e.g. a space in the variable (`PLATFORM=dev, run`)